### ISSUE TITLE
Opening Modal breaks scroll

### ISSUE THREAD

**Issue by ChrisKnightLDN** [Greater-London-Authority/ldn-viz-tools#85]
**Describe the bug**
In an app with a modal scrolling in disabled once the modal has been opened

**To Reproduce**
on an app with an open modal button and content long enough to trigger scroll
1. click open modal
2. close modal
3. try to scroll

**Expected behavior**
After the modal is closed scrolling the page should be possible

**Screenshots**
![image](https://github.com/Greater-London-Authority/ldn-viz-tools/assets/111759008/01bae031-3aba-4b16-8aa8-75652d27b7ab)
![image](https://github.com/Greater-London-Authority/ldn-viz-tools/assets/111759008/cc4f6c57-beec-4fd1-9d61-9d9aada18ca6)
![image](https://github.com/Greater-London-Authority/ldn-viz-tools/assets/111759008/8389ed12-4915-4e2f-b328-2ab6620f8404)

**Desktop (please complete the following information):**
 - OS: windows
 - Browser chrome
 - Version 114.0.5735.90

**Additional context**
A style is applied to the html on opening the modal: 
![image](https://github.com/Greater-London-Authority/ldn-viz-tools/assets/111759008/adf056b6-fc92-47c3-b73c-4b119bdefd96)
This doesn't seem to be cleared on close
**This does not seem to happen on storybook.**

**Comment by jamesscottbrown** [comment]
> This doesn't seem to be cleared on close
> This does not seem to happen on storybook.

In both storybook and the ev charging dashboard, these attributes are set on the `<html>` tag and then cleared when the modal is dismissed (this is easy to miss in storybook, since the story is inside an `iframe` so there are multiple `<html>` elements).

**Comment by ChrisKnightLDN** [comment]
Yes - sorry the comment wasn't clear - The bug is not present in storybook - because the css properties are correctly cleared when the modal is dismissed. On the effected app the css properties are not being cleared.
